.gitignore

```diff
@@ -1,2 +1,3 @@
 CMakeLists.txt.user
 .DS_Store
+build
```

Comment:
Can you please revert this? We do not want to add personalised build-related settings.

Reply:
sorry for that, i have just delete it